core/flyout_horizontal.js

```diff
@@ -32,11 +33,16 @@ goog.requireType('Blockly.utils.Metrics');
  */
 Blockly.HorizontalFlyout = function(workspaceOptions) {
   Blockly.HorizontalFlyout.superClass_.constructor.call(this, workspaceOptions);
-  
   this.horizontalLayout = true;
 };
 Blockly.utils.object.inherits(Blockly.HorizontalFlyout, Blockly.Flyout);
 
+/**
+ * The name of the horizontal flyout in the registry.
+ * @type {string}
+ */
+Blockly.HorizontalFlyout.registryName = 'horizontalFlyout';
```

Comment:
We should standardize on one name for this. This is what Rachel has in her strict-connection-checker plugin: 
https://github.com/google/blockly-samples/pull/333/files#diff-459bd1d34908031cb468abe7c18d912bR55

Not saying that's what we should go with, just that we should align.

Reply:
Ok, will keep in mind for the future. I ended up going with having flyout-vertical-toolbox and flyout-horizontal-toolbox so ended up removing this.